lib/turbograft.rb

```diff
@@ -14,16 +14,20 @@ class Config
     self.controllers = ["ActionController::Base"]
```

Comment:
If it was just me I'd remove this way to hook into the app entirely. But It's a breaking change so I haven't proposed it.

But yeah I'd be fine with this.